packages/devtools_app/lib/src/shared/console/widgets/description.dart

```diff
@@ -293,7 +295,21 @@ class DiagnosticsNodeDescription extends StatelessWidget {
       final propertyType = diagnosticLocal.propertyType;
       final properties = diagnosticLocal.valuePropertiesJson;
 
-      if (name?.isNotEmpty == true && diagnosticLocal.showName) {
+      // Show the "default" label if this is the property is the default value:
```

Comment:
nit: "if this property has the default value"

Reply:
Moved the code around, but comment and typo are gone